ci/taskcat.yml

```diff
@@ -5,8 +5,22 @@ global:
   marketplace-ami: false
   reporting: true
   regions:
+    - ap-northeast-1
```

Comment:
Because we are using `Exports` we need to limit each QuickStart test to a subset of regions. I am not particular about how the test regions are split\defined. Just need to make sure that all three products can create the services infrastructure without collisions. We can discuss this topic in greater detail for now just consider it a CI workaround.